src/pages/pt-BR/guides/server-side-rendering.md

```diff
@@ -0,0 +1,144 @@
+i18nReady: true
+---
+
+**Renderização no lado do Servidor**, também conhecido como SSR, é possível no Astro. Quando você habilita o SSR você pode:
+- Implementar sessões para um estado de login no seu app.
+- Renderizar dados de uma API chamada dinamicamente com `fetch`.
+- Fazer deploy do seu site em uma hospedagem utilizando um **adaptador**. 
+
+> SSR é novo no Astro e mudanças irão ocorrer antes do lançamento estável da v1.0. Por favor mantenha-se atualizado sobre as mudanças na API aqui.
+
+
+## Habilitando o SSR em seu Projeto
+
+Para habilitar o SSR você precisa de um adaptador. Estes são os adaptadores disponíveis hoje, com mais por vir no futuro:
+
+- [Deno](https://github.com/withastro/astro/tree/main/packages/integrations/deno)
+- [Netlify](https://github.com/withastro/astro/tree/main/packages/integrations/netlify)
+- [Node.js](https://github.com/withastro/astro/tree/main/packages/integrations/node)
+- [Vercel](https://github.com/withastro/astro/tree/main/packages/integrations/vercel)
+
+Neste exemplo iremos utilizar `@astrojs/netlify` para construir para Netlify. Primeiro, instale o adaptador:
+
+```bash
+npm install --save-dev @astrojs/netlify
+```
+
+Assim que os pacotes forem instalados, adicione duas novas linhas ao arquivo de configuração `astro.config.mjs` do seu projeto.
+
+```diff
+  // astro.config.mjs
+  import { defineConfig } from 'astro/config';
++ import netlify from '@astrojs/netlify/functions';
+
+  export default defineConfig({
++   adapter: netlify(),
+  });
+``` 
+
+Com a Netlify você pode fazer deploy pelo git, por sua interface web, ou pela interface de linha de comando (CLI). Aqui iremos utilizar a [Netlify CLI](https://docs.netlify.com/cli/get-started/) para fazer o deploy.
+
+Primeiro, construa o seu site como você faria normalmente:
+
+```bash
+npm run build
+```
+
+Isso irá criar `netlify/functions/` que contém o seu código de SSR. Fazer deploy do seu site também fará o deploy dessa função que contém todas as suas páginas Astro prontas para serem renderizadas.
+
+```bash
+netlify deploy
+```
+
+Assim que o deploy estiver completo, você receberá uma URL de demonstração do seu site.
+
+## Funcionalidades
+
+O Astro irá continuar como um gerador de sites estáticos por padrão, mas assim que você habilita um adaptador de renderização no lado do servidor, algumas novas funcionalidades são disponibilizadas a você.
+
+### Astro.request.headers
+
+Os cabeçalhos de uma requisição estão disponíveis em `Astro.request.headers`. Ele é um objeto [Headers](https://developer.mozilla.org/en-US/docs/Web/API/Headers), parecido com um Map, onde você pode pegar cabeçalhos como o cookie.
+
+```astro
+---
+const cookie = Astro.request.headers.get('cookie');
+// ...
+---
+<html>
+  <!-- Página aqui... -->
+</html>
+```
+
+### Astro.redirect
+
+Na global `Astro`, este método permite que você redirecione para outra página. Você pode fazer isso após checar se o usuário está logado pegando sua sessão de um cookie.
```

Comment:
I totally agree with the second and third changes, but is there something wrong with `Na global Astro`? Do you find it hard to understand what it means or you just think it would be better that way? I would not like to change this because we use this same translation across multiple other pages, so if we change it here, it would be inconsistent.

Reply:
I just think it  would be better that way.

Reply:
```suggestion
Na global `Astro`, este método permite que você redirecione para outra página. Você talvez faça isso após checar se o usuário está logado obtendo sua sessão de um cookie.
```